### ISSUE TITLE
Research recursive file naming #Help #Question

### ISSUE THREAD

**Issue by ldexterldesign** [laurent22/massren#36]
Hi all,

Hope you're well

Thanks for software

How would one go about renaming files recursively within a folder structure?

Since there's no features relating to this in the documentation I assume this isn't possible (yet) so I reached for (my limited) shell/bash knowledge:

``
find . -type d | xargs massren
``

Currently I only get an instance for the working directory but I expect a recursive instance per directory - can anyone make this useful?

Hope this is clear/useful and hope to hear back

Keep up great work!

Cheers

**Comment by ldexterldesign** [comment]
BUMP

**Comment by ldexterldesign** [comment]
BUMP

**Comment by laurent22** [comment]
Yes it's not currently supported. Maybe a bash file with a simple for loop would work? I don't know the xargs command well enough to know what it will do in combination with massren.

**Comment by ldexterldesign** [comment]
Cheers for reply

K, I'll post back if I find a solution

In the meantime, I'd appreciate if you left this issue open in case someone else can offer a solution

Keep up great work!

Thanks
